### PR TITLE
Create Fibonacci.cpp

### PR DESCRIPTION
**Pull request by Deadeye001**
Added C++ code which generates a sequence of first n Fibonacci numbers

### CODE REVIEW

File: C++/Fibonacci.cpp [hunk 1]
```diff
@@ -0,0 +1,35 @@
+/*
+SUBMITTED BY :- Deadeye001
+CODE TO GENERATE FIRST n FIBONACCI SEQUENCE
+*/
+#include <bits/stdc++.h>
```
Comment (DevAndromeda) [inline C++/Fibonacci.cpp:5]:
```suggestion
#include <iostream>
```

**Review by DevAndromeda — CHANGES_REQUESTED** [review]

**Comment by Deadeye001** [comment]
Done!

**Comment by DevAndromeda** [comment]
your code prints fibonacci in this sequence: `0 1 2 3 5 8 13 21 34 55`. Shouldn't that be `0 1 1 2 3 5 8 13 21 34`?

**Comment by Deadeye001** [comment]
Very sorry. Please close this PR. I'm pushing a new one with the corrected code

**Pull request merged**